packaging/suse/portusctl/lib/runner.rb

```diff
@@ -30,4 +30,30 @@ def self.activate_service(service, restart = false)
         service
       ])
   end
+
+  # Creates a new file called "versions.log" with information about the version
+  # of the different components.
+  def self.produce_versions_file!
+    docker   = `docker -v`
```

Comment:
I also thought about calling "rpm -qi docker", since rpm already handles packages that not installed with a fancy message.